### PR TITLE
Exposing more methods on wtransport::Connection from underlying quinn::Connection object

### PR DESCRIPTION
**Pull request by JustSomeDude301**
#25

### CODE REVIEW

File: wtransport/src/connection.rs [hunk 1]
```diff
@@ -114,4 +115,33 @@ impl Connection {
...
+    /// A stable identifier for this connection
+    ///
+    /// Peer addresses and connection IDs can change, but this value will remain
+    /// fixed for the lifetime of the connection.
+    #[inline(always)]
+    pub fn stable_id(&self) -> usize {
+        self.quic_connection.stable_id()
+    }
+
+    /// Compute the maximum size of datagrams that may be passed to [`send_datagram()`].
```
Comment (BiagioFesta) [inline wtransport/src/connection.rs:128]: nit: Compute(s)*

File: wtransport/src/connection.rs [hunk 2]
```diff
@@ -114,4 +115,33 @@ impl Connection {
...
+    /// estimate. The peer can also enforce an arbitrarily small fixed limit, but if the peer's
+    /// limit is large this is guaranteed to be a little over a kilobyte at minimum.
+    ///
+    /// Not necessarily the maximum size of received datagrams.
+    #[inline(always)]
+    pub fn max_datagram_size(&self) -> Option<usize> {
+        self.quic_connection.max_datagram_size()
+    }
+
+    /// Current best estimate of this connection's latency (round-trip-time)
```
Comment (BiagioFesta) [inline wtransport/src/connection.rs:142]: nit: full stop and the end of the sentence

File: wtransport/src/connection.rs [hunk 3]
```diff
@@ -114,4 +115,33 @@ impl Connection {
     pub fn remote_address(&self) -> SocketAddr {
         self.quic_connection.remote_address()
     }
+
+    /// A stable identifier for this connection
```
Comment (BiagioFesta) [inline wtransport/src/connection.rs:119]: nit: Full stop at the end of the line

**Review by BiagioFesta — CHANGES_REQUESTED** [review]
Thanks for this.

LGTM, just a few nits on doc

**Comment by BiagioFesta** [comment]
I rebased your PR and squashed in one single commit.

I merged the commit on `master`, but GitHub is not able to see the merge because I fix a nit. Can you try to rebase the branch on your fork?

**Comment by JustSomeDude301** [comment]
Rebased, please do not hesitate to let me know if you need me to do anything else in future issues. Thank you again for your suggestions on my connection map and for your work on this library. If you have a private channel (like a discord server) I'd be more than happy to share more about my use case

**Comment by BiagioFesta** [comment]
I am not really into Discord, but I think it might be a good idea to start a project related chat-community.

https://wtransport.zulipchat.com

**Pull request merged**